script/02_PoolModifyLiquidityTest.s.sol

```diff
@@ -0,0 +1,17 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.22;
+
+import {Script, console2} from "forge-std/Script.sol";
+import {IPoolManager} from "../src/interfaces/IPoolManager.sol";
+import {PoolModifyLiquidityTest} from "../src/test/PoolModifyLiquidityTest.sol";
+
+contract PoolModifyLiquidityTestScript is Script {
+
+    address poolManager = 0xc021A7Deb4a939fd7E661a0669faB5ac7Ba2D5d6;
```

Comment:
would be ideal to pass this in? maybe its best just to have 1 script called TestnetDeployScript or something where all of them get deployed and pass the pm between them